### ISSUE TITLE
Doesnt work on Samsung Galaxy S23 ultra

### ISSUE THREAD

**Issue by ZaGaZ8** [PavelDoGreat/WebGL-Fluid-Simulation#92]
Just upgraded from a OnePlus 8t and this app no longer functions correctly.
I doubt this will be taken seriously though as I have noticed the "NEW" app that has changed the charging structure - very disappointing :(  What happened to the one off "lifetime" payment?  Creating a new identical app and charging again is somewhat unfair!

**Comment by MiracleCK** [comment]
您好，我已经收到你的信件！我会及时阅读并作出回应！

**Comment by ZaGaZ8** [comment]
yeah thought so....  no intention of fixing the "old" version....  Shady tactics indeed!

**Comment by TheZitroX** [comment]
I'll guess it is about money generation these days.

**Comment by ZaGaZ8** [comment]
yeah spot on.....  This has to violate store guidelines surely?  I might do some digging :)

**Comment by mmtmn** [comment]
@samsung
